MyPiggyBank.Data/Model/CyclicOperation.cs

```diff
@@ -14,5 +14,7 @@ public class CyclicOperation
         public decimal EstimatedValue { get; set; }
         public string Period { get; set; }
```

Comment:
Zmiana typu period na int. W klasie biznesowej bedzie on enumem. String jest niebezpieczny, bo będziemy mogli do niego cokolwiek dać. Tak będzie to bardziej rygorystyczne.